jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -1341,8 +1365,14 @@ Function Start-Migration {
 
         # Conditional ParameterSet logic
         If ($PSCmdlet.ParameterSetName -eq "form") {
+            # If from form, Test-JumpCloudUsername function should be used, else JumpCloudUsername will need to be calculated here if autobind is also specified
             $SelectedUserName = $inputObject.SelectedUserName
-            $JumpCloudUserName = $inputObject.JumpCloudUserName
+            if ([system.string]::IsNullOrEmpty($JumpCloudsystemUserName)) {
```

Comment:
I'm not seeing this condition be hit as I test with the gui version of the tool

Reply:
Moved it under the $AutobindJCUser variable since it calls the Test-JumpCloudUsername which returns $JumpCloudSystemUsername